index.html

```diff
@@ -5,33 +5,45 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Marvel's Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+    
 </head>
 <body>
+    <div class="card-area" id="home">
     <header>
         <nav>
+            <img src="/images/Marvel_Logo 1.png" alt="Marvel Logo" id="marvel-logo">
             <ul>
-                <li><a href=""></a>Home</li>
-                <li><a href=""></a>About</li>
-                <li><a href=""></a>Content</li>
-                <li><a href=""></a>Contact</li>
+                <li><a href="#home"></a>HOME</li>
+                <li><a href=""></a>COMICS</li>
+                <li><a href=""></a>GAMES</li>
+                <li><a href=""></a>NEWS</li>
+                <li><button><img src="/images/search.svg" alt="searc-icon"></button></li>
```

Comment:
Try to increase this image size a bit

Reply:
Increased the size of the search button from 36.67px to 55px. 
Increased the size of the user Icon from the same size to 70px.